### PR TITLE
Add support for setting/overriding config from envvars

### PR DESCRIPTION
**Pull request by AntonOfTheWoods**
This is practical for environments like docker/kubernetes

### CODE REVIEW

File: ankisyncd/config.py [hunk 1]
```diff
@@ -11,6 +11,16 @@
...
+# Get values from ENV and update the config. To use this prepend `ANKISYNCD_`
+# to the uppercase form of the key. E.g, `ANKISYNCD_SESSION_MANAGER` to set
+# `session_manager`
+def load_from_env(conf):
+    logging.info("Loading/overriding config values from ENV")
+    for env in os.environ:
+        if env.startswith('ANKISYNCD_'):
+            config_key = env[10:].lower()
+            conf[config_key] = os.getenv(env)
+            logging.info(f"Setting {config_key} from ENV")
```
Comment (tsudoko) [inline ankisyncd/config.py]: F-strings are Python 3.6+, the server currently still supports Python 3.5 so you'll need to change this.

File: ankisyncd/config.py [hunk 2]
```diff
@@ -11,6 +11,16 @@
     os.path.join(dirname(dirname(realpath(__file__))), "ankisyncd.conf"),
 ]
 
+# Get values from ENV and update the config. To use this prepend `ANKISYNCD_`
+# to the uppercase form of the key. E.g, `ANKISYNCD_SESSION_MANAGER` to set
+# `session_manager`
+def load_from_env(conf):
+    logging.info("Loading/overriding config values from ENV")
```
Comment (tsudoko) [inline ankisyncd/config.py]: Seems a bit too verbose

Comment (AntonOfTheWoods) [reply ankisyncd/config.py]: Changed to debug

File: ankisyncd/config.py (hunk 1 above)
Comment (AntonOfTheWoods) [reply ankisyncd/config.py]: Changed to `.format`

**Pull request merged**